src/components/table/Table.vue

```diff
@@ -160,6 +160,7 @@
 </template>
 
 <script>
+/* eslint-disable */
```

Comment:
Sorry. I had to do it to be able to run my dev server without eslint complaining. I've already remove it.